src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1161,7 +1161,28 @@ class WorkspaceService(protected val userInfo: UserInfo, dataSource: SlickDataSo
 
   def workflowQueueStatus() = {
     dataSource.inTransaction { dataAccess =>
-      dataAccess.workflowQuery.countWorkflowsByQueueStatus.map(RequestComplete(StatusCodes.OK, _))
+      dataAccess.workflowQuery.countWorkflowsByQueueStatus.flatMap { statusMap =>
+
+        // determine the current size of the workflow queue
+        val numQueued = statusMap.getOrElse(WorkflowStatuses.Queued.toString, 0)
```

Comment:
I was able so simplify from `x:Int` to `x` but that's it. The `Some()` is invalid because getOrElse doesn't return an `Option`.